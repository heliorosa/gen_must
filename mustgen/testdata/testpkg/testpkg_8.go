package testpkg

func (t3 TypeC[T, U]) otherMethod() (int, error) {
	//@gen_must
	return 0, nil
}
