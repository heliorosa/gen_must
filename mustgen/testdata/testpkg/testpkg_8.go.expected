// Code generated - DO NOT EDIT.
// This file is auto generated by gen_must and any manual changes will be lost.

package testpkg

// mustOtherMethod has the behavior of otherMethod, except it panics on error
func (t3 TypeC[T, U]) mustOtherMethod() int {
	var0, err := t3.otherMethod()
	if err != nil {
		panic(err)
	}
	return var0
}
